text/metrics/0108-naming-conventions.md

```diff
@@ -0,0 +1,23 @@
+# Metric instrument naming conventions
```

Comment:
this document reads more like naming guidelines, not conventions. Conventions would actually include recommendations for common patterns, like `runtime.***`.

Reply:
#119 is coupled with this document with some conventions. https://github.com/open-telemetry/opentelemetry-specification/pull/657 describes a conventional naming pattern.

Originally this document contained conventions, and we split the work apart. @tedpennings would you substitute "guidelines" in for "conventions"?

Reply:
Changed to guidelines in 8b88b36